main.go

```diff
@@ -127,7 +128,12 @@ func findNums(prefix string, kvs map[string]interface{}, out io.Writer) {
 			continue
 		}
 
-		_, err := fmt.Fprintf(out, "rsyslog.%v.%v:%d|g\n", prefix, sanitizeKey(k), int(vf))
+		hostname, err := os.Hostname()
+		if err != nil {
+			el.Println("Error determining hostname", err)
+		}
+
+		_, err = fmt.Fprintf(out, hostname+".rsyslog.%v.%v:%d|g\n", prefix, sanitizeKey(k), int(vf))
```

Comment:
thinking about this a bit, but I wonder if I need to modify the hostname

`os.Hostname()` should return something like `db1.sjc`, but that `.` will cause issues and needs to be converted to `-`, right?

Reply:
Yeah the `.` will cause another level of nesting, so you will indeed need to replace it with a `-`

Reply:
Come to think of it, I would put `rsyslog` first and then the hostname so they are easier to find by drilling down